app/support/minutes_to_time_formatter.rb

```diff
@@ -0,0 +1,21 @@
+class MinutesToTimeFormatter
+  attr_reader :minutes
+
+  def initialize(minutes)
+    @minutes = minutes
+  end
+
+  def to_s
+    "#{hours}:#{padded_minutes}"
+  end
+
+  private
+
+  def hours
+    (minutes / 60).floor
+  end
+
+  def padded_minutes
+    (minutes % 60).to_s.rjust(2, '0')
```

Comment:
Neat. I haven't seen `rjust` used much (I tend to run immediately toward `sprintf`/`format` and not look back).